src/theory/arith/nl/transcendental/proof_checker.cpp

```diff
@@ -26,9 +26,83 @@ namespace arith {
 namespace nl {
 namespace transcendental {
 
+namespace {
+
+/**
+ * Evaluates the first d terms of the Maclaurin series for the exponential
+ * function at t. This is a lower bound on the exponential function.
+ */
+inline Node evalMaclaurinExp(std::uint64_t d, TNode t)
```

Comment:
Can we make this code shared between the solver and the proof checker? (Similar questions for the other utility methods).  This is a principle I've been trying to stick to, that the proof checker and the solver should use common utilities.